.env.dev

```diff
@@ -11,6 +11,12 @@ REACT_APP_REDUX_WINDOW=false
 # mixpanel
 REACT_APP_MIXPANEL_TOKEN=a867ce40912a6b7d01d088cf62b0e1ff
 
+# gatefi
```

Comment:
these aren't required in `.env.dev` or `.env.develop` if they're present and identical in `.env.base`